genai-perf/genai_perf/inputs/input_retriever_factory.py

```diff
@@ -78,12 +78,6 @@ def get_input_data(self) -> Dict:
             )
         else:
             if self.config.input_type == PromptSource.DATASET:
-                # (TMA-1990) support VLM input from public dataset
```

Comment:
Moved this check to the input type check function in inputs.py. We can start moving all similar checks (output_format vs input_type) there to centralize those checks.

Reply:
Thanks for the heads up! I can remove this if there ends up being a conflict.